GoRogue/MapGeneration/Region.cs

```diff
@@ -332,15 +325,15 @@ public virtual Region Rotate(double degrees, Point origin)
         /// Adds a sub-region to this region.
         /// </summary>
         /// <param name="region">The region you wish to add as a sub-region</param>
-        public void Add(Region region)
+        public void AddRegion(Region region)
```

Comment:
Why `AddRegion` instead of `AddSubRegion?`

Reply:
No particular reason. Changed in new version